metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
OK.  This is important.  A parser developer who is developing and maintaining their own parsers outside the metron tree should not have to makes changes to the metron code tree.  Ever.

As an extension developer, I want my extension to be a manageable, singular entity.  The packaging should contain everything todo with my extension.   I would ask back, why when I create an extension would I **not** have all the configurations and bits together?